.setup/install_submitty/get_globals.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+set -ve
+
+# First, we determine SUBMITTY_REPOSITORY from BASH_SOURCE, as this
+# script is run from within GIT_CHECKOUT/Submitty. We use readlink to
+# get an absolute path from a relative path here.
+SUBMITTY_REPOSITORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/../.."
+SUBMITTY_REPOSITORY="$(readlink -ne "$SUBMITTY_REPOSITORY")"
+echo "SUBMITTY_REPOSITORY=${SUBMITTY_REPOSITORY}" >&2
+
+SUBMITTY_INSTALL_DIR="$(readlink -ne "${SUBMITTY_REPOSITORY}/../..")"
```

Comment:
this line doesn't make sense to me.  We should not assume this relationship between these 2 directories.

Reply:
Hmmm... I see the problem but this is a little bit tricky to deal with. Right now, SUBMITTY_INSTALL_HELPER and company are run from within the repo instead of within the install dir. After looking through the rest of the code in this PR, this relationship is also assumed in this relative path in SUBMITTY_INSTALL_HELPER:
https://github.com/Submitty/Submitty/blob/787d35a2da9f669c6cbb6668aca9d557057857ba/.setup/INSTALL_SUBMITTY_HELPER.sh#L36
So we need a better way to deal with this. One thing we could do is copy scripts from the repo to the install dir before SUBMITTY_INSTALL_HELPER is run. Alternatively we can export a variable containing the location of the install dir before calling SUBMITTY_INSTALL_HELPER. I think copying scripts is probably a better way to do this, so I will work on that.